gtfs/spec/en/reference.md

```diff
@@ -411,6 +432,9 @@ To process the cost of a leg:
 | `network_id` | Foreign ID referencing `routes.network_id` | Optional | Identifies a route network that applies for the fare leg rule.<br><br>If there are no matching `fare_leg_rules.network_id` values to the `network_id` being filtered, empty `fare_leg_rules.network_id` will be matched by default.<br><br> An empty entry in `fare_leg_rules.network_id` corresponds to all networks defined in `routes.txt` excluding the ones listed under `fare_leg_rules.network_id` |
 | `from_area_id` | Foreign ID referencing `areas.area_id` | Optional | Identifies a departure area.<br><br>If there are no matching `fare_leg_rules.from_area_id` values to the `area_id` being filtered, empty `fare_leg_rules.from_area_id` will be matched by default. <br><br>An empty entry in `fare_leg_rules.from_area_id` corresponds to all areas defined in `areas.area_id` excluding the ones listed under `fare_leg_rules.from_area_id` |
 | `to_area_id` | Foreign ID referencing `areas.area_id` | Optional | Identifies an arrival area.<br><br>If there are no matching `fare_leg_rules.to_area_id` values to the `area_id` being filtered, empty `fare_leg_rules.to_area_id` will be matched by default.<br><br> An empty entry in `fare_leg_rules.to_area_id` corresponds to all areas defined in `areas.area_id` excluding the ones listed under `fare_leg_rules.to_area_id` |
+|  `start_timeframe_group_id` | Foreign ID referencing `timeframes.timeframe_group_id` | Optional |  Defines the timeframe for the departure of the fare leg.<br><br>For a fare leg rule that specifies a `start_timeframe_group_id`, that rule will match a particular leg if there exists at least a row in `timeframes.txt` where all of the following conditions are true:<br>- The row’s `timeframe_group_id` is equal to the `start_timeframe_group_id` value.<br>- The set of service days identified by the row’s `service_id` contains the service day of the leg’s departure trip.<br>- The departure time of the leg, relative to the service day per [GTFS Time field type](#field-types) conventions, is greater than or equal to the rows `start_time` value and less than the `end_time` value.<br><br>If there are no matching `fare_leg_rules.start_timeframe_group_id` values to the `timeframe_group_id` being filtered, empty `fare_leg_rules.start_timeframe_group_id` will be matched by default. <br><br>An empty `fare_leg_rules.start_timeframe_group_id` indicates that the start time of the leg does not affect the fare. |
```

Comment:
if there exists at least a row in `timeframes.txt`

I noticed you added "at least" here.  Did you mean to say "if there exists at least one row"?  Or something different?

Reply:
You're right, I added "at least", because we have had more than one row. 

I changed 

> if there exists at least a row in `timeframes.txt`

to:

> if there exists at least one row in `timeframes.txt`

Is this better?